submissions/unabyband/task-js-dom/script.js

```diff
@@ -0,0 +1,58 @@
+//DOM functions here
+
+const mainContent = {
```

Comment:
`mainContent` should store data not ids or html. Also, `mainContent` is an unclear name.

Reply:
Done and done.